File: python/8 kyu/206.py
# https://www.codewars.com/kata/57f780909f7e8e3183000078/train/python

# Passed

def grow(arr):
    result = 1
    for num in arr:
        result *= num
        
    return result

output = grow([2, 2, 2, 2, 2, 2])
print(output)

File: nodejs/7 kyu/206.js
// https://www.codewars.com/kata/5a8d1c82373c2e099d0000ac/train/javascript

// Passed

function solve(st, a, b) {
  let result =
    st.slice(0, a) +
    st
      .slice(a, b + 1)
      .split("")
      .reverse()
      .join("") +
    st.slice(b + 1);
  return result;
}

const output = solve("codewars", 1, 5);
console.log(output);